_episodes/05-plotting.md

```diff
@@ -114,3 +131,152 @@ plt.savefig('all_samples.png')
 >> {: .language-python}
 > {: .solution}
 {: .challenge}
+
+### Plotting with `x` and `y`
+
+The `plot` function creates a line plot. If there is only one argument in the function, it will use that as as the `y` variable, with the `x` variable just being a count.
+
+If we wanted to use the `Frame` column for our `x` value, we would add it as the first argument of the `plot` function. This won't change the graph at all.
+
+```
+import matplotlib.pyplot as plt
+
+plt.figure()
+plt.plot(data[:,0], data[:,1])
+```
+{: .language-python}
+
+However, if we did not want to plot every frame, or if the data we were plotting was not sequential, we would have to specify `x` and `y`.
+
+Let's try plotting every 100th frame. To do this we will use a new slicing syntax. To select frames at regular intervals, we can use the syntax `array[start:stop:interval]`.
+
+For example, to get every other row to the 10th row, we would use a start of `0`, and end of `10`, and an increment of `2`. We then use the `:` to get every column.
+
+~~~
+data[0:10:2, :]
+~~~
+{: .language-python}
+
```

Comment:
Given our emphasis on immutable data, I think this might confuse the students.  You aren't changing data here.  I would probably save this slice as a new variable name or at least highlight that you aren't changing data by taking this slice.

Reply:
Would adding a `print` in front be better?

Reply:
Yeah, I think so.  I just think that this notation might make students think it changed data, and that's exactly what we don't want them to think.